tool/clean/clean_eks/clean_eks.go

```diff
@@ -16,6 +16,9 @@ import (
 	"github.com/aws/amazon-cloudwatch-agent/tool/clean"
 )
 
+// make this configurable or construct based on some configs?
+const betaEksEndpoint = "https://api.beta.us-west-2.wesley.amazonaws.com"
+
 var (
 	ClustersToClean = []string{
 		"cwagent-eks-integ-",
```

Comment:
Correct me if I'm wrong but is the prefix for beta cluster `cwagent-operator-eks-integ`? If so, should we add that pattern here?